bindings/python/iota_sdk/wallet/account.py

```diff
@@ -223,7 +223,7 @@ def addresses_with_unspent_outputs(
         addresses = self._call_account_method(
             'addressesWithUnspentOutputs'
         )
-        return [from_dict(AddressWithUnspentOutputs, address)
+        return [AddressWithUnspentOutputs.from_dit(address)
```

Comment:
from_dict*

Reply:
still wrong

Reply:
No?

Reply:
hm odd,i guess i redirected to outdated link of file